noria-server/src/controller/migrate/mod.rs

```diff
@@ -333,16 +492,11 @@ impl<'a> Migration<'a> {
             }
         }
         let swapped = swapped0;
+
+        // Assign local addresses to all new nodes, and initialize them.
         let mut sorted_new = new.iter().collect::<Vec<_>>();
```

Comment:
You may want to add a `map` here so that this becomes a `Vec<NodeIndex>`. That way you don't have to have all the `&&&&&&&&` below :p

Reply:
I also wonder if we should just filter on `!is_dropped()` here, since afaict we add that filter every time we use `sorted_new` below.

Reply:
Was this actually resolved? I think you are still missing a `map(|&ni| ni)`?

Reply:
👍